src/EthereumConnection.js

```diff
@@ -1,15 +1,19 @@
-require("./utils/env.js")();
 const log = require("./utils/log.js")("EthereumConnection");
 const EventEmitter = require("events");
 const promiseTimeout = require("./utils/promiseTimeout.js");
 const setExitHandler = require("./utils/sigintHandler.js");
 const Web3 = require("web3");
 
 // TODO: make these a DEFAULT_OPTIONS = {...} object and all options to EthereumConnection.options = { ...}
-const DEFAULT_ETHEREUM_CONNECTION_TIMEOUT = 10000;
-const DEFAULT_ETHEREUM_CONNECTION_CLOSE_TIMEOUT = 10000;
-const DEFAULT_ETHEREUM_ISLISTENING_TIMEOUT = 1000; // used at isConnected() for web3.eth.net.isListening() timeout. TODO: check if we still need with newer web3 or better way?
-const DEFAULT_ETHEREUM_CONNECTION_CHECK_INTERVAL = 1000;
+const DEFAULTS = {
```

Comment:
you essentially implemented the TODO thanks! we can remove the comment line above :)